src/hotspot/share/prims/vectorSupport.cpp

```diff
@@ -430,6 +430,18 @@ int VectorSupport::vop2ideal(jint id, BasicType bt) {
       }
       break;
     }
+    case VECTOR_OP_MASK_TOLONG: {
+      switch (bt) {
+        case T_BYTE:  // fall-through
+        case T_SHORT: // fall-through
+        case T_INT:   // fall-through
+        case T_LONG:  // fall-through
+        case T_FLOAT: // fall-through
+        case T_DOUBLE: return Op_VectorMaskToLong;
+        default: fatal("MASK_TRUECOUNT: %s", type2name(bt));
```

Comment:
MASK_TOLONG here? Also break is missing.

Reply:
This is consistent with other case statements.

Reply:
default: fatal("MASK_TRUECOUNT: %s", type2name(bt));
should refer to MASK_TOLONG.